### ISSUE TITLE
Regression: does not work with aliases in an Astro project

### ISSUE THREAD

**Issue by tex0l** [tachibana-shin/vite-plugin-arraybuffer#10]
**Describe the bug**
I have a resource I want to import as `arraybufer` which is located within the directory `./src/resources` from my base directory, which is aliased using the `tsconfig.json` configuration [like it is mandated by Astro](https://docs.astro.build/en/guides/imports/#aliases):

```json
{
  /* ... */
  "compilerOptions": {
    /* ... */
    "baseUrl": ".",
    "paths": { "~/*": [ "src/*" ] },
  }
}
```

I import it from a file with:

```ts
import myResource from '~/resources/my-resource.png?arraybuffer'
```

**To Reproduce**
See the stackblitz here: https://stackblitz.com/edit/github-5tpg4s?file=src%2Fpages%2Findex.astro

Try to `pnpm install && pnpm start`, it'll fail with the error `ENOENT: no such file or directory, open '~/resources/my-resource.png'`

If you roll back to 0.0.2, it'll work.

**Expected behavior**
I would expect the alias to be respected like in 0.0.2.

**Additional context**
Somehow, it works with Astro 3 with vite-plugin-arraybuffer 0.0.7, but not with Astro 4.

**Comment by tachibana-shin** [comment]
Astro has upgraded their vite so a few APIs have changed. I will fix it tomorrow

**Comment by tachibana-shin** [comment]
Fixed on `vite-plugin-arraybuffer@0.0.8`
